mvc/system/classes/frameWork.php

```diff
@@ -1,20 +1,26 @@
 <?php
 class frameWork
 {
-
 	public function view($viewName)
```

Comment:
Preferable to use two spaces for indentation

Reply:
changes done .